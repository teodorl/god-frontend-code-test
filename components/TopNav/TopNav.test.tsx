import React from 'react';
import { render } from '../../test-utils';
import { TopNav } from './';

describe('<TopNav /> spec', () => {
    it('renders the component', () => {
        const container = render(<TopNav />);
        expect(container.firstChild).toMatchSnapshot()
    });
});
