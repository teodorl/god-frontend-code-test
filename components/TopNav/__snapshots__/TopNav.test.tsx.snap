// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TopNav /> spec renders the component 1`] = `undefined`;
